ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift

```diff
@@ -121,7 +121,8 @@ open class FileDisplayScreenlet: BaseScreenlet {
 		interactor.cacheStrategy = CacheStrategyType(rawValue: self.offlinePolicy ?? "") ?? .remoteFirst
 
 		interactor.onSuccess = {
-			if let resultAsset = interactor.asset {
+			if let resultAsset = interactor.asset, let assetMimeType = resultAsset.mimeType,
+					self.supportedMimeTypes.contains(assetMimeType) {
 				self.fileEntry = FileEntry(attributes: resultAsset.attributes)
```

Comment:
Some error message to the console? because if the mimetype is not supported it silently fails, right?